samples/tools/autogenbench/scenarios/GAIA/Scripts/demystifying-telemetry-sqlite.ipynb

```diff
@@ -0,0 +1,425 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demystifying telemetry.sqlite artifact\n",
+    "\n",
+    "When you run logging enabled, a db file will be generated in sqlite format.\n",
+    "\n",
+    "See https://microsoft.github.io/autogen/docs/Use-Cases/enhanced_inference/#logging for more info.\n",
+    "\n",
+    "In this notebook we will only use two tables, chat_completions and events."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prereq"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install jsondiff\n",
+    "%pip install ipywidgets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load database, and do some data wrangling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3\n",
+    "\n",
+    "target_sqlite_db = r\"C:\\webarena__reddit_TwoAgents\\69\\0\\telemetry.sqlite\"\n",
```

Comment:
Can you remove hard coded paths?

Reply:
yeah - I left it as a clue for anyone that wants to plug-in their own sqlite db. I'll clean up once this PR is not wip/draft.